src/components/v5/common/ActionSidebar/partials/UserSelect/hooks.ts

```diff
@@ -1,28 +1,51 @@
+import { useMemo } from 'react';
 import { useColonyContext } from '~hooks';
 import { useGetColonyMembers } from '~v5/shared/MembersSelect/hooks';
 import { UserSelectHookProps } from './types';
+import { useGetVerifiedMembersQuery } from '~gql';
+import { SearchSelectOption } from '~v5/shared/SearchSelect/types';
 
-export const useUserSelect = (): UserSelectHookProps => {
+export const useUserSelect = (inputValue: string): UserSelectHookProps => {
   const { colony } = useColonyContext();
-  const { members, loading } = useGetColonyMembers(colony?.colonyAddress);
+  const { colonyAddress = '', metadata } = colony ?? {};
+  const { members, loading } = useGetColonyMembers(colonyAddress);
+  const { data, loading: verifiedMembersLoading } = useGetVerifiedMembersQuery({
+    variables: { colonyAddress },
+    skip: !colonyAddress,
+  });
+  const isWhitelistActivated = metadata?.isWhitelistActivated;
 
-  const users = members?.map((member) => {
-    const { label, walletAddress, avatar } = member || {};
+  const verifiedUsers: SearchSelectOption[] = useMemo(
+    () =>
+      (data?.getContributorsByColony?.items ?? []).map((item, index) => ({
+        value: item?.user?.profile?.displayName,
```

Comment:
The value should never be `diplayName` as it's not a good indicator of uniqueness. Use `walletAddress` or id instead

Reply:
I know but when I'll change it on `walletAddress` then for sec next to user avatar instead of user name will be displayed wallet address and then will be replaced by user name if exist